src/crashhandler_windows.cpp

```diff
@@ -171,6 +171,12 @@ namespace g3 {
          raise(signal_number);
       }
 
+      // FYI: Concept of async-signal-safe operations does not exist on windows
+      // we stick to perror for lack of better alternatives.
+      size_t writeErrorMessage(const char* message) {
+         perror(message)
```

Comment:
semicolon missing -- it sucks that there's no easy way to spin up vscontainer with windows afaik